dockerfiles/MXNetContainerDockerfile

```diff
@@ -0,0 +1,20 @@
+ARG CODE_VERSION
+FROM clipper/py-rpc:${CODE_VERSION}
+
+COPY clipper_admin/clipper_admin/python_container_conda_deps.txt /lib/
+
+RUN echo deb http://ftp.de.debian.org/debian jessie-backports main >> /etc/apt/sources.list \
+      && apt-get update --fix-missing \
+      && apt-get install -yqq -t jessie-backports openjdk-8-jdk \
+      && conda install -y --file /lib/python_container_conda_deps.txt \
+      && pip install mxnet==1.0.0 \
+      && apt-get install -y graphviz \
+      && pip install graphviz \
+      && conda install -c anaconda cloudpickle=0.5.2
+
+
+COPY containers/python/mxnet_container.py containers/python/mxnet_container_entry.sh containers/python/train.data /container/
```

Comment:
Remove the `containers/python/train.data` file from the container

Reply:
Done.